src/app/static/src/app/store/projects/actions.ts

```diff
@@ -117,6 +120,16 @@ export const actions: ActionTree<ProjectsState, RootState> & ProjectsActions = {
             }
         }
     },
+    
+    async updateVersionNote(context, versionPayload: versionPayload) {
+        const {projectId, versionId} = versionPayload.version
+        const note = versionPayload.note
+
+        await api<ProjectsMutations, ProjectsMutations>(context)
+            .withSuccess(ProjectsMutations.SetPreviousProjects)
```

Comment:
OK I see that you are committing this response - see above remark on using existing pattern to fetch projects in a separate request